csharp/tools/Microsoft.ML.OnnxRuntime.PerfTool/Microsoft.ML.OnnxRuntime.PerfTool.csproj

```diff
@@ -33,7 +33,7 @@
   <ItemGroup>
     <ProjectReference Include="$(OnnxRuntimeCSharpRoot)\src\Microsoft.ML.OnnxRuntime\Microsoft.ML.OnnxRuntime.csproj" />
     <PackageReference Include="CommandLineParser" Version="2.4.3" />
-    <PackageReference Include="Microsoft.ML.Scoring" Version="1.1.0" />
+    <PackageReference Include="Microsoft.ML.OnnxRuntime" Version="0.3.1" />
```

Comment:
0.3.1?

Reply:
The perftool project needs to be updated in general (i.e. read from protobuf input data files). 

@hariharans29 -since we have a reference to the csproj (below), we should be able to drop this package reference now.

Include="$(OnnxRuntimeCSharpRoot)\src\Microsoft.ML.OnnxRuntime\Microsoft.ML.OnnxRuntime.csproj" />

Reply:
Oh that's right. So let me just drop this line in the csproj. Thanks.